superset-frontend/src/dashboard/components/RefreshIntervalModal.tsx

```diff
@@ -91,6 +91,52 @@ class RefreshIntervalModal extends React.PureComponent<
     this.setState({
       refreshFrequency: value || refreshIntervalOptions[0][0],
     });
+
+    const custom_block = document.getElementById('custom_block_view');
```

Comment:
is it possible to control/select this element with a ref instead of directly fetching from the dom?

Reply:
while implementing this, i was not getting how to hide/unhide custom options from react so i implemented onclick to hide/unhide dom element.

Not sure how can i do without dom element, but i will check and do if possible.